corehq/ex-submodules/dimagi/utils/couch/undo.py

```diff
@@ -72,4 +72,4 @@ def undo_delete(document, save=True):
 
 
 def remove_deleted_doc_type_suffix(doc_type):
-    return doc_type.rstrip(DELETED_SUFFIX)
+    return doc_type.removesuffix(DELETED_SUFFIX)
```

Comment:
Have all environment migrated to Python 3.9?

Reply:
According to https://dimagi.github.io/commcare-cloud/changelog/0045-upgrade-to-python-39.html they have to be.

Reply:
Oh that is very cool!